pkg/controller/bootstrap/testdata/bootstrap/machineconfigcontroller-controllerconfig.yaml

```diff
@@ -32,6 +32,9 @@ spec:
       controlPlaneTopology: HighlyAvailable
       platformStatus:
         type: None
+  dns:
```

Comment:
Just curious, was this added for a test? I can't seem to match what this is being used for

Reply:
some of the vsphere templates need the dns field to render properly